GitUI/LeftPanel/RepoObjectsTree.SettingsContextMenu.cs

```diff
@@ -85,6 +85,15 @@ private void ReorderTreeNode(TreeNode node, bool up)
             ShowEnabledTrees();
         }
 
+        public void EmptyTrees()
```

Comment:
To align with the .NET Framework and .NET vocabulary
```suggestion
        public void ClearTrees()
```

Reply:
Done.